libs/cardano-data/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Version history for `cardano-data`
 
+## 1.1.0.0
+
+* Fix - A bug was fixed in the `canonicalInsert` function.
+  The bug manifested by creating an unbalanced tree in the `Data.Map` internals of the
+  'CanonicalMap', which can result in a crash.
```

Comment:
Maybe adding a reference to the issue that describes the crash in the changelog would be a good thing to do?
https://github.com/input-output-hk/cardano-node/issues/4826